go/cmd/vttestserver/main.go

```diff
@@ -56,37 +55,37 @@ var (
 	topo      topoFlags
 )
 
-func init() {
-	flag.IntVar(&basePort, "port", 0,
+func RegisterFlags(fs *pflag.FlagSet) {
```

Comment:
nit: no need to export this, as it's a `main` package no one is going to import it

Reply:
Fixed